pandas/core/frame.py

```diff
@@ -3161,6 +3167,21 @@ def query(self, expr, inplace=False, **kwargs):
         For further details and examples see the ``query`` documentation in
         :ref:`indexing <indexing.query>`.
 
+        *Backtick quoted variables*
+
+        Backtick quoted variables are parsed as literal Python code and
+        are converted internally to a Python valid identifier.
+        This can lead to the following problem.
+
+        The parsing can result in an error for certain characters.
+        These characters are those that fall outside the ASCII range
+        (U+0001..U+007F), but also the hashtag (as it used for comments)
```

Comment:
are there tests for when backquoating *doesn't* work?

Reply:
There was one, but I added two more.